include/cling/Interpreter/Interpreter.h

```diff
@@ -468,6 +481,19 @@ namespace cling {
     ///
     CompilationResult parseForModule(const std::string& input);
 
+    ///\brief Code completes user input.
+    ///
+    /// The interface circumvents the most of the extra work necessary to
+    /// code complete code.
+    ///
+    /// @param[in] input - The input containing the string to be completed.
+    /// @param[out] T - The cling::Transaction of the input
+    ///
+    ///\returns Whether the operation was fully successful.
+    ///
+    CompilationResult codeComplete(const std::string& input, unsigned offset,
+                                   Transaction** T = 0);
```

Comment:
We don't need the transaction, we don't expect anything valuable as result, given that this might not even be parsed